_data/projects/kodadot.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: KodaDot - Polkadot NFT marketplace
+desc: open source implementation of nine NFT standards and growing to become multi-chain available nft marketplace
+site: http://github.com/kodadot/nft-gallery
+tags:
+- polkadot
+- vuejs
+- vue3
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ForNeVeR/Cesium/labels/good%20first%20issue
```

Comment:
🤔 this should be a label in your repository, not the Cesium project

Reply:
Hey, you did this commit apparently by accident? https://github.com/up-for-grabs/up-for-grabs.net/pull/4108/commits/520cf215370eb9fbbbd1ee42b972017184a41cd4